### ISSUE TITLE
Add way to visualise the 'visible' and 'invisible' sides of the limb

### ISSUE THREAD

**Issue by dstansby** [sunpy/sunpy#5540]
I'm just working on https://github.com/sunpy/sunpy/pull/5414, which allows one to plot the limb from an arbitrary observer:
![Screenshot 2021-09-24 at 15 37 51](https://user-images.githubusercontent.com/6197628/134693047-dedcd60f-e83e-49ff-86c5-6fd43af73ffb.png)

From just a single line it's not clear which side of the limb is the visible surface, and which is the invisible surface. It would be really nice to have some way of indicating which one is which. I'm not sure how at the moment, but putting this out there in case anyone has any ideas!

**Comment by ayshih** [comment]
Here's less structure in the indicators:
![download (40)](https://user-images.githubusercontent.com/991759/136393944-81ed9804-5f9c-4dd3-80d1-bbf95ed68cd0.png)
![download (41)](https://user-images.githubusercontent.com/991759/136393978-f0a27516-df58-40c0-b529-62fad48ac03f.png)

**Comment by Cadair** [comment]
This should be closed?

**Comment by ayshih** [comment]
> This should be closed?

Why?

**Comment by nabobalis** [comment]
Oh that would explain why I can't find a PR for this.

**Comment by wtbarnes** [comment]
Is this not closed by #6251? Reading the original issue, it seems like these lines https://github.com/sunpy/sunpy/blob/5aaa19df333dfab9be8c5ebe52d35a950ca402b7/sunpy/visualization/drawing.py#L272-L284 solve the problem.

**Comment by ayshih** [comment]
Nope, this issue is not closed.  This issue is about visually indicating which "half" of the (2D) Sun's surface is not visible, not merely which "half" of the (1D) solar limb is not visible.
